src/main/scala/game/GameEngine.scala

```diff
@@ -15,6 +15,24 @@ class GameEngine {
 
   def generatePirate = generatePlayer(3)
 
+  /**
```

Comment:
Thx! I'm going to make these changes as soon as I get some time and we can then discuss next month